packages/adapters-library/src/config.ts

```diff
@@ -215,10 +218,11 @@ function parseNumberEnv(env: string | undefined, defaultValue: number): number {
 }
 
 function parseStringEnv(env: string | undefined, defaultValue: string): string {
-  // Uses defaultValue if empty string is passed
-  return env || defaultValue
+  // Uses defaultValue only if env is undefined
+  return env ?? defaultValue
```

Comment:
This is only used for the provider names. Doing this, if we don't pass any provider URL, it'll use the default as it'll be undefined, but if we pass an empty string we can then disable that provider without the need for another environment variable (handy for BSC and Fantom without having to comment code).